From 8813654d23b0b06ccb0cca9cc9d7956ec17682e6 Mon Sep 17 00:00:00 2001
From: David Holsgrove <david.holsgrove@petalogix.com>
Date: Mon, 8 Oct 2012 16:09:58 +1000
Subject: [PATCH 40/41] microblaze: Define UTIME_NOW and UTIME_OMIT in stat.h

Required in libmount for util-linux and missing in microblaze header

Signed-off-by: David Holsgrove <david.holsgrove@petalogix.com>
---
 sysdeps/unix/sysv/linux/microblaze/bits/stat.h |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/sysdeps/unix/sysv/linux/microblaze/bits/stat.h b/sysdeps/unix/sysv/linux/microblaze/bits/stat.h
index 822d948..76bd102 100644
--- a/sysdeps/unix/sysv/linux/microblaze/bits/stat.h
+++ b/sysdeps/unix/sysv/linux/microblaze/bits/stat.h
@@ -192,4 +192,9 @@ struct stat64
 #define __S_IWRITE      0200    /* Write by owner.  */
 #define __S_IEXEC       0100    /* Execute by owner.  */
 
+#ifdef __USE_ATFILE
+# define UTIME_NOW	((1l << 30) - 1l)
+# define UTIME_OMIT	((1l << 30) - 2l)
+#endif
+
 #endif	/* bits/stat.h */
-- 
1.7.1

